### ISSUE TITLE
Publish templates?

### ISSUE THREAD

**Issue by jadjoubran** [jadjoubran/laravel-ng-artisan-generators#2]
I was thinking maybe we can publish the templates as well in the `resources/generators` so that users can have the flexibility of modifying the default template
What do you think?

**Comment by m33ch** [comment]
This is a great idea!

So would you migrate stub files to jadjoubran/laravel5-angular-material-starter ?

**Comment by jadjoubran** [comment]
Yes we probably need to publish the stubs to jadjoubran/laravel5-angular-material-starter
I have a card that on the original repo that will remove the stubs and original functionality
so we can then include this once I implement the recipes system

**Comment by m33ch** [comment]
Ok @jadjoubran 

Need change each command for get contents of stub file.

Maybe it usefull to set in config file source directory of generators. 
What do you think?

**Comment by jadjoubran** [comment]
That's great!

**Comment by m33ch** [comment]
Hi @jadjoubran,

namespace has been changed.

**Comment by jadjoubran** [comment]
Awesome great!
I'll release a 1.0 version as soon as possible (hopefully this weekend)
